modules/bitcoinj/app/src/main/kotlin/bip32/master_key_gen/Wrapper.kt

```diff
@@ -0,0 +1,32 @@
+package bip32.master_key_gen
+
+import org.bitcoinj.crypto.DeterministicKey
+import org.bitcoinj.crypto.HDKeyDerivation
+import org.bitcoinj.base.BitcoinNetwork
+
+object MasterKeyGenerator {
```

Comment:
This object could be more generic
```suggestion
object BitcoinJWrapper {
```